frontend/src/pages/AccountSend.vue

```diff
@@ -36,7 +66,13 @@
 
       <!-- Send button -->
       <div>
-        <base-button :disable="isSending" :full-width="true" label="Send" :loading="isSending" type="submit" />
+        <base-button
+          :disable="!isValidForm || isSending"
+          :full-width="true"
+          label="Send"
+          :loading="isSending"
+          type="submit"
+        />
         <base-button
           @click="generatePaymentLink({ to: recipientId, token, amount: humanAmount })"
```

Comment:
Payment links with amounts seem to be broken. Try visiting http://localhost:8080/send?to=benrink.eth&token=ETH&amount=1 as an example. The console shows an error message:

<img width="837" alt="Screen Shot 2021-10-05 at 1 22 56 PM" src="https://user-images.githubusercontent.com/1479802/136075592-a9cc4eda-a441-43db-af07-7fdb7f74f5d4.png">